packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerForm/AddPrayerForm.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
I got a lot of warnings when I clicked on the first story. Is this all from Formik?

![Screenshot 2019-04-18 13 34 54](https://user-images.githubusercontent.com/3229463/56379887-cb1fbe80-61de-11e9-82e4-dec717108b28.png)

Reply:
yes that's all from formik but if you yarn'd you should have gotten the updated `formik` that gets rid of those warnings

Reply:
This is after I yarn'd. So ... closer. Would this be need an update to `Switch` then? I'm guessing yes and if so, we could probably just ignore it for now. Although this might be worth fixing in TAP core so that they go away ... sometime down the road. 

![Screenshot 2019-04-18 14 09 26](https://user-images.githubusercontent.com/3229463/56381685-982bf980-61e3-11e9-81ec-4d67dd9b2997.png)

Reply:
...and yes I have an open core PR to fix the `Switch` :)